packages/gatsby-theme-docs/src/modules/ai-assistant/components/chat-modal.jsx

```diff
@@ -40,9 +45,9 @@ const LEGAL_DISCLAIMER = (
   <>
     Results are generated using generative AI and should be verified prior to
     use. By interacting with the feature, you agree to our “
-    <Link href="/../offering/ai-disclaimer" target="_blank">
+    <a href="/../offering/ai-disclaimer" target="_blank">
```

Comment:
Why don't you want to use gatsby's Link component here?

Reply:
> Why don't you want to use gatsby's Link component here?

[Gatsby Link API](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/)
It has no `href` and `target` props. It still works probably because internally it falls back to `a` but it creates a bunch of warnings in the console.

Reply:
@timonrey  We're either using our own Link wrapper component or pure <a> tags , we never use the gatsby link directly.   If possible, use our link wrapper (it's not flexible enough at the moment, I know - very monolithic and hard to adapt to differing styling and such).   Here it's not a big deal because it's a new window and a hardwired microsite.